### DISCUSSION TITLE
HMS-800-2T als 2. WR findet keine Verbindung zur openDTU

### DISCUSSION THREAD

**Discussion by Fokker-1** [tbnobody/OpenDTU#2012] [Q&A]
Hallo,

ich betreibe seit einem Jahr eine HM-800-2T  (Typ ohne "S", ohne "W") an der openDTU, das funktioniert mit absoluter Zufriedenhei.t. Mein Dank an die SW-Entwickler hier!

Nun habe ich mir einen weiteren WR vom Typ HM**S**-800-2T (ohne "W") zugelegt und wollte den als 2. WR in die die openDTU einbinden.

Dazu habe ich unter "Wechselrichter Einstellungen / neuen Wechselrichter hinzufügen" dessen Seriennummer (die wurde bestätigt als Hoymiles SN) und meinen gewünschten Namen dieses 2. WR eingetragen und gespeichert- mehr Einträge habe ich nicht gemacht.

Nun wird auf dem Startscreen der openDTU auch dieser 2. WR angezeigt mit korrektem Namen.
Aber im blauen Kreis befindet sich ein Ausrufezeichen, kein Häkchen. Dieser 2. WR wird also nicht erkannt, es gibt auch keine empfangene Daten.

Mehrfaches Ein- und Ausschalten des WR und / oder der openDTU brachten keine Veränderung.
Der ursprüngliche HM-800 ist weiterhin mit korrekten Anzeigen verfügbar.

Die Entfernung des HMS-800 zur openDTU beträgt ca. 2 m im gleichen Raum.

Ich bin nun ratlos! Muss man ggf. noch weitere Einstellungen zur Einbindung dieses 2. WR in die openDTU machen?

Für hilfreiche Imformationen bin ich sehr dankbar!

Viele Grüße
Werner

**Comment by tbnobody** [comment]
Für einen HMS Inverter benötigst du ein anderes Funkmodul (andere Frequenz Features etc.). Siehe Übersichtsliste: https://www.opendtu.solar/hardware/inverter_overview/

**Comment by Fokker-1** [comment]
Hallo,

sorry für meine späte Rückmeldung  - Termine, Termine ...

Vielen Dank für die hilfreiche Info, daran hatte ich nicht gedacht.
Ich habe mir jetzt eine Fusion-DTU bestellt, die kann die HM- und auch HMS-Systeme, das sollte dann klappen.
Leider ist der Preis dafür dann schon etwas höher.

Viele Grüße
Werner
